unit_tests/test_utils_data_validator.py

```diff
@@ -0,0 +1,94 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2022 ScyllaDB
+import logging
+import os
+import unittest
+
+from sdcm.utils.data_validator import LongevityDataValidator
+from sdcm import sct_config
+
+
+class ConfigurationBase(unittest.TestCase):  # pylint: disable=too-many-public-methods
+    _testMethodName = "runTest"
+
+    @classmethod
+    def setUpClass(cls):
+        logging.basicConfig(level=logging.ERROR)
+        logging.getLogger('botocore').setLevel(logging.CRITICAL)
+        logging.getLogger('boto3').setLevel(logging.CRITICAL)
+        logging.getLogger('anyconfig').setLevel(logging.ERROR)
+
+        cls.setup_default_env()
+
+        os.environ['SCT_CLUSTER_BACKEND'] = 'aws'
+        os.environ['SCT_AMI_ID_DB_SCYLLA'] = 'ami-06f919eb'
+        cls.conf = sct_config.SCTConfiguration()
```

Comment:
what's this for ? if each test is creating it's own configuration ?

Reply:
changed